bindata/network/ovn-kubernetes/alert-rules-control-plane.yaml

```diff
@@ -48,3 +48,21 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: V4SubnetAllocationThresholdExceeded
+      annotations:
+        message: |
+          More than 80% of v4 subnets available to assign to the nodes are allocated
```

Comment:
You should add templating to the message (see openshift-sdn/alert-rules.yaml) that displays the actual percentage allocated.

Reply:
Done.